kodilla-hibernate/src/test/java/com/kodilla/hibernate/tasklist/dao/TaskListDaoTestSuite.java

```diff
@@ -28,8 +28,9 @@ public void testFindByListName() {
         taskListDao.save(taskList);
         //When
         List<TaskList> readListName = taskListDao.findByListName(LIST_NAME);
+        String taskListName = taskList.getListName();
         //Then
-        Assert.assertEquals(LIST_NAME, readListName);
+        Assert.assertEquals(LIST_NAME, taskListName);
```

Comment:
Teraz asercję robisz na obiekcie, który utworzyłeś w linii 30. To nie jest obiekt pobrany z bazy danych!
Wystarczy:
`Assert.assertEquals(LIST_NAME, readListName.get(0).getListName());`
Jest tylko jeden element o zadanej nazwie, więc możesz bez problemu jawnie się do niego odnieść.